pango/sys/src/lib.rs

```diff
@@ -843,19 +866,18 @@ impl ::std::fmt::Debug for PangoGlyphGeometry {
 }
 
 #[repr(C)]
-#[derive(Copy, Clone)]
 pub struct PangoGlyphInfo {
     pub glyph: PangoGlyph,
     pub geometry: PangoGlyphGeometry,
-    pub attr: PangoGlyphVisAttr,
+    _truncated_record_marker: c_void,
+    // /*Ignored*/field attr has incomplete type
```

Comment:
Meh this change is problematic, I wonder why...

Reply:
Yes that's bad. You need to know the proper struct layout for being able to make it boxed inline.

What changed here in the headers for it to be completely broken now? :)